browser/extensions/brave_component_loader.cc

```diff
@@ -108,6 +109,14 @@ void BraveComponentLoader::AddDefaultComponentExtensions(
       brave_webtorrent_path.Append(FILE_PATH_LITERAL("brave_webtorrent"));
     Add(IDR_BRAVE_WEBTORRENT, brave_webtorrent_path);
   }
+
+#if BUILDFLAG(BRAVE_WALLET_ENABLED)
+  // If brave://wallet has been loaded at least once, then load it again.
+  if (ExtensionPrefs::Get((content::BrowserContext *)profile_)->
+    HasPrefForExtension(ethereum_remote_client_extension_id)) {
```

Comment:
lint - 4 spaces

Reply:
fixed, didn't trigger lint though. I think the 4 spaces on line continuations within conditions ins't enforced I guess.